tests/core/command/command.js

```diff
@@ -302,5 +302,30 @@ bender.test( {
 		editor._.elementsPath.onClick( 0 );
 
 		wait();
+	},
+
+	'test addCommand from command instance': function() {
```

Comment:
It'd be good to add test using `CKEDITOR.command` subclass, just in case to test if inherited commands are treated the same.